src/api/notification/email-templates/marriage/registration.html

```diff
@@ -37,7 +37,7 @@
                 max-width: 100%;">
     <h1>Marriage registration completed</h1>
     <p>
-        Hello {{informantName}},
+        Hello{{formatName informantName}},
     </p>
     <p>
         Congratulations, the marriage of {{groomName}} and {{birdeName}} has been registered.
```

Comment:
```
Congratulations, the marriage of {{groomName}} and {{birdeName}} has been registered.
```

`birdeName` must be a bug here as well

Reply:
I see, I think we should test out all the email templates with empty value to find out what the users experiences in corner cases. On it.

Reply:
On another note, only in the in progress email, there is a chance that values can be null. And in progress emails have only one variable, that is `informantName`. So, I guess only handling null value `informantName` should suffice.